### PR TITLE
Add missing React and Vue code snippets to dxDataGrid.sortByGroupSummaryInfo

### PR DESCRIPTION
**Pull request by onebalaban**

### CODE REVIEW

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]
onToolbarPreparing examples somehow sneaked into this PR.

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 1]
```diff
@@ -68,6 +68,76 @@ Normally, when records are grouped by a column, the groups are sorted according
...
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid>
+            <DxSummary>
+                <DxGroupItem 
+                    column="Age"
+                    summaryType="avg"
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    summary-type="avg"
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 2]
```diff
@@ -68,6 +68,76 @@ Normally, when records are grouped by a column, the groups are sorted according
...
+                    name="Average Age Group Summary"
+                />
+                <DxGroupItem
+                    column="Income"
+                    summaryType="max"
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    summary-type="max"
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 3]
```diff
@@ -68,6 +68,76 @@ Normally, when records are grouped by a column, the groups are sorted according
...
+                />
+                <DxGroupItem
+                    column="Tasks"
+                    summaryType="min"
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    summary-type="min"
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 4]
```diff
@@ -114,7 +184,58 @@ To use these summary items for sorting groups, assign an array of objects to the
...
         // ...
     })
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid>
+            <DxSortByGroupSummaryInfo :summary-item="1"/> <!-- determines the maximum income item using its index in the "groupItems" array -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
            <DxSortByGroupSummaryInfo
                :summary-item="1"/> <!-- determines the maximum income item using its index in the "groupItems" array -->
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 5]
```diff
@@ -114,7 +184,58 @@ To use these summary items for sorting groups, assign an array of objects to the
...
+            <DxSortByGroupSummaryInfo summary-item="min"/> <!-- determines the minimum tasks item using its aggregate function -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
            <DxSortByGroupSummaryInfo
                summary-item="min"/> <!-- determines the minimum tasks item using its aggregate function -->
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 6]
```diff
@@ -114,7 +184,58 @@ To use these summary items for sorting groups, assign an array of objects to the
...
+            <DxSortByGroupSummaryInfo summary-item="Average Age Group Summary"/> <!-- determines the average age item using its name -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
            <DxSortByGroupSummaryInfo
                summary-item="Average Age Group Summary"/> <!-- determines the average age item using its name -->
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 7]
```diff
@@ -114,7 +184,58 @@ To use these summary items for sorting groups, assign an array of objects to the
...
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DataGrid, SortByGroupSummaryInfo } from 'devextreme-react/data-grid';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <DataGrid>
+                    <SortByGroupSummaryInfo summaryItem={1}/> {/* determines the maximum income item using its index in the "groupItems" array */}
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    <SortByGroupSummaryInfo
                        summaryItem={1}/> {/* determines the maximum income item using its index in the "groupItems" array */}
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 8]
```diff
@@ -114,7 +184,58 @@ To use these summary items for sorting groups, assign an array of objects to the
...
+                    <SortByGroupSummaryInfo summaryItem="min"/> {/* determines the minimum tasks item using its aggregate function */}
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    <SortByGroupSummaryInfo
                        summaryItem="min"/> {/* determines the minimum tasks item using its aggregate function */}
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 9]
```diff
@@ -114,7 +184,58 @@ To use these summary items for sorting groups, assign an array of objects to the
...
+                    <SortByGroupSummaryInfo summaryItem="Average Age Group Summary"/> {/* determines the average age item using its name */}
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    <SortByGroupSummaryInfo
                        summaryItem="Average Age Group Summary"/> {/* determines the average age item using its name */}
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 10]
```diff
@@ -164,6 +285,69 @@ After that, set the [groupColumn](/api-reference/10%20UI%20Widgets/dxDataGrid/1%
...
+        <DxDataGrid>
+            <DxSortByGroupSummaryInfo
+                :summary-item="1"
+                group-column="Tasks"
+            /> <!-- applies sorting only when records are grouped by the "Tasks" column -->
+            <DxSortByGroupSummaryInfo
+                summary-item="min"
+                group-column="Last Name"
+            /> <!-- applies sorting only when records are grouped by a "Last Name" column -->
+            <DxSortByGroupSummaryInfo summary-item="Average Age Group Summary"/> <!-- applies sorting regardless the grouping column -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
            <DxSortByGroupSummaryInfo
                summary-item="Average Age Group Summary"
            /> <!-- applies sorting regardless the grouping column -->
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md [hunk 11]
```diff
@@ -164,6 +285,69 @@ After that, set the [groupColumn](/api-reference/10%20UI%20Widgets/dxDataGrid/1%
...
+                <DataGrid>
+                    <SortByGroupSummaryInfo
+                        summaryItem={1}
+                        groupColumn="Tasks"
+                    /> {/* applies sorting only when records are grouped by the "Tasks" column */}
+                    <SortByGroupSummaryInfo
+                        summaryItem="min"
+                        groupColumn="Last Name"
+                    /> {/* applies sorting only when records are grouped by a "Last Name" column */}
+                    <SortByGroupSummaryInfo summaryItem="Average Age Group Summary"/> {/* applies sorting regardless the grouping column */}
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/sortByGroupSummaryInfo/sortByGroupSummaryInfo.md]:
```suggestion
                    <SortByGroupSummaryInfo
                        summaryItem="Average Age Group Summary"
                    /> {/* applies sorting regardless the grouping column */}
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — APPROVED** [review]

**Review by artem-kurchenko — APPROVED** [review]

**Pull request merged**